### PR TITLE
Examples: update webgl_materials_cars

### PR DESCRIPTION
**Pull request by looeee**
[Live version](https://rawgit.com/looeee/three.js/examples_car_controls/examples/webgl_materials_cars.html) 

webgl_materials_car

As discussed in #13651: 

* combined `webgl_materials_cubemap_dynamic` and `webgl_materials_cars` so that there is now one place that demonstrates car controls with car.js and dynamic material updating. 
* removed dependency on `BinaryLoader` from `Car.js` so that it now accepts a generic car model in any format
* Replaced the old car models with [this](https://sketchfab.com/models/57bf6cc56931426e87494f554df1dab6) one, converted to glTF and with draco compression it comes in at 1.6mb. 
* Renamed `webgl_materials_cubemap_dynamic2` -> `webgl_materials_cubemap_dynamic` and removed the old example. 

After this PR there should be nowhere that uses the `BinaryLoader`, actually deprecating it and tidying old files is left to a seperate PR.

### CODE REVIEW

File: examples/webgl_materials_cars.html [hunk 1]
```diff
@@ -66,541 +61,338 @@
...
 
-					$( button_name( car, i ) ).counter = i;
-					$( button_name( car, i ) ).addEventListener( 'click', function() {
+				carModel.getWorldPosition( cameraTarget );
+				cameraTarget.y = 2.5;
+        cameraTarget.z += distance;
 
-						for ( var j = 0; j < material_indices.length; j ++ ) {
+				camera.position.lerp( cameraTarget, delta * damping );
+        camera.lookAt( carModel.position );
```
Comment (Mugen87) [inline examples/webgl_materials_cars.html]: There are some minor format issue at the end of the file.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: examples/webgl_materials_cars.html [hunk 2]
```diff
@@ -66,541 +61,334 @@
...
+      }
 
-						}
-
-					}, false );
-
-				}
-
-			}
+      function resetPosition() {
```
Comment (Mugen87) [inline examples/webgl_materials_cars.html]: Um, this looks still not correct. BTW: You can easily detect wrong formatting by enabling invisible characters in your editor.

Comment (looeee) [reply examples/webgl_materials_cars.html]: Thanks, should all be fixed now.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by donmccurdy** [comment]
Looks great! 🎉

**Pull request merged**